docs/source/quickstart.rst

```diff
@@ -44,34 +45,37 @@ If models are already saved to file, they can also be brought to BentoML with th
     clf.fit(X, y)
 
     bentoml.sklearn.save("iris_clf", clf)
-    # [INFO] Scikit-learn model 'iris_clf:yftvuwkbbbi6zcphca6rzl235' is successfully saved to BentoML local model store under "~/bentoml/models/iris_clf/yftvuwkbbbi6zcphca6rzl235"
+    # [08:34:16 AM] INFO     Successfully saved Model(tag="iris_clf:svcryrt5xgafweb5",
+    #                        path="/home/user/bentoml/models/iris_clf/svcryrt5xgafweb5/")
+    # Tag(name='iris_clf', version='svcryrt5xgafweb5')
 
-The :ref:`ML framework specific API <frameworks-page>`, `bentoml.sklearn.save()`, will save the Iris Classifier to a 
-local model store managed by BentoML. And the `load_runner()` API can be used to load this model into a Runner:
+The :ref:`ML framework specific API <frameworks-page>`, `bentoml.sklearn.save()`, will save the Iris
+Classifier to a local model store managed by BentoML. You can then load the runner for this model
```

Comment:
```suggestion
Classifier to a local model store managed by BentoML. Users can then load the runner for this model
```

Reply:
I'd say 2nd person is quite common in docs, so I think this is fine.